helm/istio/charts/tracing/templates/jaeger-all-in-one.yaml

```diff
@@ -25,9 +25,6 @@ spec:
       {{- end }}
 
   agent:
```

Comment:
maybe remove the whole agent node

Reply:
was wondering about that - but thought may be for consistency between the templates, just to show the component could be configured. But if others also think not relevant can remove.